decidim-verifications/lib/decidim/verifications/sms/engine.rb

```diff
@@ -18,7 +18,7 @@ class Engine < ::Rails::Engine
           root to: "authorizations#new"
         end
 
-        initializer "decidim.sms_verification_workflow" do |_app|
+        initializer "decidim_verifications_sms.sms_verification_workflow" do |_app|
```

Comment:
Maybe the name of this initializer is too redundant.... 
maybe a name like `decidim_verifications_sms.workflow` would be more appropriate?

Reply:
There were also many other places where we had such redundancy. First I thought I'd handle that separately since the idea of this was just to fix the inconsistency with the prefixes across different engines.

But anyways, I fixed those in this same PR at the same time. Fixed by ea41f97501850bc12d95774018f14e8ba02b74e7.